### PR TITLE
clarifying PSP role and binding

### PR DESCRIPTION
**Pull request by puja108**
also need to fix numbered lists in example chapters

### CODE REVIEW

**Comment by puja108** [comment]
@oponder @marians any idea how to keep the numbers but not make HUGO translate that to a numbered list (and always start with `1.`)?

**Comment by marians** [comment]
@puja108 I can't see any numbers

**Comment by oponder** [comment]
@puja108 the text has to be indented so that it doesn't think it is a new block that should 'interupt' the list.

Made a commit. Lets see if our markdown interpreter correctly displays it now.

File: content/guides/securing-with-rbac-and-psp/index.md [hunk 1]
```diff
@@ -209,11 +209,13 @@ By default, Giant Swarm clusters come with two PSPs defined - a `privileged` pol
 
 Additionally, there are two respective cluster roles defined - `privileged-psp-user` and `restricted-psp-user`.
 
-By default all authenticated users are assigned the `restricted-psp-user` role.
+By default all authenticated users are bound the `restricted-psp-user` role.
```
Comment (oponder) [inline content/guides/securing-with-rbac-and-psp/index.md]: grammar: shoudl be "bound to the"

File: content/guides/securing-with-rbac-and-psp/index.md [hunk 2]
```diff
@@ -209,11 +209,13 @@ By default, Giant Swarm clusters come with two PSPs defined - a `privileged` pol
...
 
-If you need to run privileged containers, you need to either use a cluster admin user or bind the `privileged-psp-user` role to your desired user or group.
+If you need to run privileged containers in a Pod, you need to either use a cluster admin user or bind the `privileged-psp-user` role to your desired user or group.
 
-Keep in mind that Pods that need extensive Security Contexts need to run with a [Service Account](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/) that has a similarly extensive PSP bound to it. You can either bind the already existing `privileged-psp-user` role to it, or create a PSP that caters specifically to your desired Security Context.
+Note thatour user's PSP only come into play when you directly create Pods. If you are using Deployment, Daemonsets, or other means to spawn Pods in your cluster, you need to make sure these have the right PSP by using [Service Accounts](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/).
```
Comment (oponder) [inline content/guides/securing-with-rbac-and-psp/index.md]: type: "thatour"

Comment (oponder) [inline content/guides/securing-with-rbac-and-psp/index.md]: grammar: should be "using Deployments" or possibly "using a Deployment"

File: content/guides/securing-with-rbac-and-psp/index.md [hunk 3]
```diff
@@ -209,11 +209,13 @@ By default, Giant Swarm clusters come with two PSPs defined - a `privileged` pol
...
+
+You need to either bind the already existing `privileged-psp-user` role to it, or create a PSP that caters specifically to your desired Security Context. Keep in mind that after creating a PSP you also need to have a role that is allowed to `use` that specifc PSP and bind that to the Service Account you're using. For an example see [Running Applications that need Privileged Access](#running-applications-that-need-privileged-access).
```
Comment (oponder) [inline content/guides/securing-with-rbac-and-psp/index.md]: What is 'it' in this case? The service account? I lost context due to new paragraph and am unsure what we are referring to.

File: content/guides/securing-with-rbac-and-psp/index.md (hunk 1 above)
Comment (puja108) [reply content/guides/securing-with-rbac-and-psp/index.md]: isn't it the other way around? you bind a role to a user, not a user to a role

Comment (oponder) [reply content/guides/securing-with-rbac-and-psp/index.md]: not sure about that. I'm just saying the grammar is a bit off

Comment (puja108) [reply content/guides/securing-with-rbac-and-psp/index.md]: fixed

File: content/guides/securing-with-rbac-and-psp/index.md (hunk 3 above)
Comment (puja108) [reply content/guides/securing-with-rbac-and-psp/index.md]: changed to be more clear

**Comment by puja108** [comment]
@oponder PTAL

**Pull request merged**